debezium-core/src/main/java/io/debezium/relational/Selectors.java

```diff
@@ -188,17 +189,33 @@ public TableSelectionPredicateBuilder excludeSchemas(String schemaNames) {
          * 
          * @param fullyQualifiedTableNames the comma-separated list of fully-qualified table names to include; may be null or
          *            empty
+         * @param tableIdConverter an arbitrary converter used to convert TableId into String for pattern matching.
+         *         Usually used to remove a component from tableId to simplify patterns.
          * @return this builder so that methods can be chained together; never null
          */
-        public TableSelectionPredicateBuilder includeTables(String fullyQualifiedTableNames) {
+        public TableSelectionPredicateBuilder includeTables(String fullyQualifiedTableNames, Function<TableId, String> tableIdConverter) {
             if (isEmpty(fullyQualifiedTableNames)) {
                 tableInclusions = null;
             } else {
-                tableInclusions = Predicates.includes(fullyQualifiedTableNames, TableId::toString);
+                tableInclusions = Predicates.includes(fullyQualifiedTableNames, tableIdConverter);
```

Comment:
Looking at its usage, it probably should be rather `tableIdElementConverter` or `tableIdPartConverter`?